homeassistant/components/lyric/climate.py

```diff
@@ -181,6 +188,13 @@ def temperature_unit(self) -> str:
         """Return the unit of measurement."""
         return self._temperature_unit
 
+    @property
+    def precision(self) -> float:
+        """Return the precision of the system."""
+        return (
+            PRECISION_WHOLE if self.device.units == "Fahrenheit" else PRECISION_HALVES
+        )
```

Comment:
This can be set once in the entity constructor, by setting `self._attr_precision`.

Reply:
Done!